zerostash/src/commands/commit.rs

```diff
@@ -21,6 +21,7 @@ impl AsyncRunnable for Commit {
     async fn run(&self) {
         let mut stash = self.stash.open();
         stash.load_all().unwrap();
+        migration(&mut stash);
```

Comment:
Shouldn't migration run every time we open a stash? As such, might make more sense to move the migration call to `Stash::open`. Would this break anything? Technically migration shouldn't auto-commit, so this would be safe, right?

Reply:
I think this would break the code because the files are not loaded into memory when opening the stash